src/web/server.rkt

```diff
@@ -0,0 +1,161 @@
+#lang racket
+
+(require openssl/sha1)
+
+(require "../sandbox.rkt" "../config.rkt" "../syntax/read.rkt")
+(require (submod "../timeline.rkt" debug))
+
+(provide completed-job? get-results-for get-improve-job-data job-count
+ is-server-up create-job start-job is-job-finished wait-for-job
+ start-job-server)
+
+#| Job Server Public API section |#
+
+; Helers to isolated *completed-jobs*
+(define (completed-job? id) 
+ (hash-has-key? *completed-jobs* id))
+
+(define (get-results-for id) 
+ (hash-ref *completed-jobs* id))
+
+; I don't like how specific this function is but it keeps the API boundary.
```

Comment:
Yeah, I agree & I also agree with the decision here.

Reply:
Any suggestions to remove or break down into better API calls? Or don't worry about it for now.

Maybe surfacing some of this data in a timeline API will help make a better abstraction for this.

Reply:
Eh, I don't like anything depending on the timeline. If nothing reads it, you can't break things by writing to it. So I think the API is mostly fine.